starter-code/functions-and-arrays.js

```diff
@@ -20,67 +20,117 @@ var words = [
   "orchard",
   "crackpot"
 ];
+
+function findLongestWord(words) {
+    var num=0;
+    var long = "";
+    for (var i=0; i< words.length; i++){
+      if (words[i].length>num){
+        num=words[i].length;
+        long=words[i];
+      }
+  }
+return long;
+}
 var longest = findLongestWord(words);
 console.log(longest);
+// crocodile
 
-// Calculating a Sum
-function sumArray (array) {
 
-}
 
+// Calculating a Sum
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+
+function sumArray(array){
+  var sum = 0;
+for (var i=0; i<array.length; i++){
+  sum = sum + array[i];
+}
+return sum;
+}
 var total = sumArray(numbers);
 console.log(total);
 
+
+
+
 // Calculate the Average
-function averageNumbers (array) {
+var numbers = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function averageNumbers(array){
+  var avg = total/array.length;
+  return avg;
 }
 
-var numbers = [2, 6, 9, 10, 7, 4, 1, 9];
+function sumArray(array){
+  var sum = 0;
+for (var i=0; i<array.length; i++){
+  sum = sum + array[i];
+}
+return sum;
+}
+
+var total = sumArray(numbers);
+
 var average = averageNumbers(numbers);
 console.log(average);
 
-// Array of Strings
-function averageWordLength (array) {
 
-}
 
+
+// Array of Strings
 var words = [
-  "seat",
-  "correspond",
-  "linen",
-  "motif",
-  "hole",
-  "smell",
-  "smart",
-  "chaos",
-  "fuel",
-  "palace"
+	"seat",
+	"correspond",
+	"linen",
+	"motif",
+	"hole",
+	"smell",
+	"smart",
+	"chaos",
+	"fuel",
+	"palace"
 ];
+
+function averageWordLength(array){
+  var sum = 0;
+  var avg = 0;
+  for (var i=0; i<array.length; i++){
+  sum = sum + array[i].length;
+  avg = sum/array.length;
+  }
+  return avg;
+}
+
 var averageLength = averageWordLength(words);
 console.log(averageLength);
 
-// Unique Arrays
-function uniquifyArray (array) {
 
-}
 
+
+// Unique Arrays
 var words = [
-  "crab",
-  "poison",
-  "contagious",
-  "simple",
-  "bring",
-  "sharp",
-  "playground",
-  "poison",
-  "communion",
-  "simple",
-  "bring"
-];
-var uniqued = uniquifyArray(words);
-console.log(uniqued);
+   "crab",
+   "poison",
+   "contagious",
+   "simple",
+   "bring",
+   "sharp",
+   "playground",
+   "poison",
+   "communion",
+   "simple",
+   "bring"
+ ];
+
+ var words2 = [];
```

Comment:
This is the solution. However you should but it in the body of the `doesWordExist`function.